cmssw-tool-conf.spec

```diff
@@ -164,6 +163,7 @@ Requires: freetype-toolfile
 Requires: tkonlinesw-toolfile
 Requires: py2-cx-oracle-toolfile
 Requires: oracle-toolfile
+Requires: openloops-toolfile
```

Comment:
this is not correct , we should not add openloop for amd64 archs. better to define
%define isslc %(case %{cmsos} in (slc*) echo 1 ;; (*) echo 0 ;; esac)

on top and then use to to add openloops requirement.